README.markdown

```diff
@@ -36,7 +36,8 @@ Guard::Yard also provides some basic options for doc generation and running the
 
 Available options:
 
-    :server => true         # Disable/Enable server
+    :enable_server => true  # Disable/Enable server. Defalt true
+    :server => "webrick"    # Specify which server to use
```

Comment:
You can already specify additional command line options for the YARD server using `cli`:

```
cli: '-s webrick'
```

See below, in this option list, for an example.

Reply:
I ... have made a mistake.

Reply:
No problem! Thanks for opening the PR anyways. Contributions are always welcome.